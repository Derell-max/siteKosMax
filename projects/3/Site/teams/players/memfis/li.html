<html>
 <head>
 <meta charset="Windows-1251">
 <title>Кортни Ли</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/memfis.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/memfis/6.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">77</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">151</font></td>
     <td><font face="impact" color="white">2</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white">74</font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">157</font></td>   
     <td><font face="impact" color="white">2</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">2354:27</font></td>
     <td><font face="impact" color="white">30:35</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">21</font></td>
     <td><font face="impact" color="white">0.3</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">777</font></td>
     <td><font face="impact" color="white">10.1</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">178</font></td> 
     <td><font face="impact" color="white">2.3</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">198/419</font></td>
     <td><font face="impact" color="white">2.6/5.4</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">75</font></td> 
     <td><font face="impact" color="white">1</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">47.3%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">78</font></td>  
     <td><font face="impact" color="white">1</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">90/225</font></td>
     <td><font face="impact" color="white">1.2/2.9</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">12</font></td>
     <td><font face="impact" color="white">0.2</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">40%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">18</font></td>
     <td><font face="impact" color="white">0.2</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">111/129</font></td>
     <td><font face="impact" color="white">1.4/1.7</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">129</font></td>   
     <td><font face="impact" color="white">1.7</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Кортни Ли</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">86%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">594</font></td>  
     <td><font face="impact" color="white">7.7</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>