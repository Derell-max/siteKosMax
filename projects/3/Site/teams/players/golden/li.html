<html>
 <head>
 <meta charset="Windows-1251">
 <title>Дэвид Ли</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/golden.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/golden/13.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">49</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">86</font></td>
     <td><font face="impact" color="white">1.8</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white">4</font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">176</font></td>   
     <td><font face="impact" color="white">3.6</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">904:21</font></td>
     <td><font face="impact" color="white">18:27</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">81</font></td>
     <td><font face="impact" color="white">1.7</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">388</font></td>
     <td><font face="impact" color="white">7.9</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">257</font></td> 
     <td><font face="impact" color="white">5.2</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">160/311</font></td>
     <td><font face="impact" color="white">3.3/6.3</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">31</font></td> 
     <td><font face="impact" color="white">0.6</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">51.4%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">49</font></td>  
     <td><font face="impact" color="white">1</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">0/2</font></td>
     <td><font face="impact" color="white">0/0</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">26</font></td>
     <td><font face="impact" color="white">0.5</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">0%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">21</font></td>
     <td><font face="impact" color="white">0.4</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">68/104</font></td>
     <td><font face="impact" color="white">1.4/2.1</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">83</font></td>   
     <td><font face="impact" color="white">1.7</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Дэвид Ли</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">65.4%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">446</font></td>  
     <td><font face="impact" color="white">9.1</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>